tests/10apidoc/00prepare.pl

```diff
@@ -26,3 +28,19 @@ sub do_request_json_for
       %args,
    );
 }
+
+
+sub new_User
```

Comment:
Rather than create this as a function you could just use the `-named_constructor` form; see https://metacpan.org/pod/Struct::Dumb#CONSTRUCTOR-FORMS